models/java/gbfs-java-model/pom.xml

```diff
@@ -0,0 +1,506 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.mobilitydata</groupId>
+    <artifactId>gbfs-java-model</artifactId>
+    <version>3.1.3-SNAPSHOT</version>
+
+    <name>gbfs-java-model</name>
+
+    <description>Generates Java model from GBFS JSON schema using jsonschema2pojo.</description>
+    <url>https://github.com/MobilityData/gbfs-json-schema</url>
+    <organization>
+        <name>MobilityData</name>
+        <url>https://mobilitydata.org/</url>
+    </organization>
+    <licenses>
+        <license>
+            <name>EUPL-1.2 with modifications</name>
+            <url>https://joinup.ec.europa.eu/software/page/eupl</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
```

Comment:
I don't mind staying on this list, but you might want to add someone at MobilityData as well. Notice that this is needed for maven central.